src/components/MapView/DateSelector/index.tsx

```diff
@@ -27,6 +27,9 @@ import {
 import { months, getMonthStartAndEnd, isAvailableMonth } from './utils';
 import { AvailableDates } from '../../../config/types';
 
+// Set default timezone to UTC
```

Comment:
Do it make sense to set that in `src/components/App/index.tsx`?

Reply:
Let's have you play around with that when fixing #30